notebooks/book2/15/logreg_iris_bayes_2d.ipynb

```diff
@@ -1,49 +1,229 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        return logprob_prior + logprob_likeli
</pre><p>I would rename to <code>log_prior</code> and <code>log_lik</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done</p>